css/style.css

```diff
@@ -0,0 +1,84 @@
+@media only screen and (min-width: 1px) {
+  body {
+    background-color: #303841;
+    color: #f0f0f0;
+    margin: 0;
+    font-family: 'RaleWay', sans-serif;
+  }
+}
+
+html {
+    padding: 0;
+}
+
+h1 {
+    font-size: 60px;
+}
+
+#navbar {
+background-color: rgb(190,49,68);
+  display: flex;
+  align-items: center;
+  position: sticky;
+  top: 0px;
+  justify-content: flex-end;
+  margin: 0;
+  border-bottom: solid black 2px;
+  height: 65px;
+}
+
+#navbar::before {
+  content: "";
+  position: absolute;
+  top: 0px;
+  left: 0px;
+  height: 100%;
+  width: 100%;
+  z-index: -1;
+  opacity: 0.7;
+}
+
+#navbar ul {
+  display: flex;
+  gap: 60px;
+  
+}
+#navbar ul li {
+  list-style: none;
+  font-size: 1.3rem;
+  margin: 16px;
+  
+}
+
+#navbar ul li a {
+  color: white;
+  display: block;
+  padding: 10%;
+  border-radius: 20px;
+  
+  text-decoration: none;
+}
+
+#navbar ul li :hover{
+    background:blue;
+    border: 1px solid #d0d0d0;
+   
+   
+}
```

Comment:
Good job so far making the navbar links show a blue background when hovered over. Per the requirements of the project, please cover the entirety of the link's text with a blue background so that it appears as displayed in the project's reference page. Almost there!

- [ ]